lgc/patch/PatchDescriptorLoad.cpp

```diff
@@ -243,7 +243,11 @@ Value *PatchDescriptorLoad::getDescPtrAndStride(ResourceNodeType resType, unsign
 
     // We need to change the stride to 4 dwords. It would otherwise be incorrectly set to 12 dwords
     // for a sampler in a combined texture.
-    stride = builder.getInt32(DescriptorSizeSampler);
+    if (node->type == ResourceNodeType::DescriptorYCbCrSampler)
```

Comment:
Could be a more concise way:)
stride = builder.getInt32(node->type == ResourceNodeType::DescriptorYCbCrSampler ? DescriptorSizeSamplerYCbCr :  DescriptorSizeSampler);